travis/stretch/push_debs_github.sh

```diff
@@ -0,0 +1,10 @@
+echo "PUSHING TO GITHUB PAGES"
+if [ `git branch --list ${PLATFORM} `]
+then
+    echo "Branch ${PLATFORM} already exists, deleting"
+fi
+git checkout --orphan ${PLATFORM}
+git rm -rf .
+git add results/
+git commit -m "pyqrllib ${PLATFORM} release"
+git push https://randomshinichi:$GITHUB_TOKEN@github.com/randomshinichi/qrllib.git HEAD:${PLATFORM} -f
```

Comment:
destination repo points to randomshinichi. It is better if this is an environment variable, otherwise we will need a second PR and this will not work.